src/test/unit/telemetry_mavlink_unittest.cc

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Thanks for adding the unit test!

I think this will also need an entry in `src/test/Makefile`, so that it gets run when `make test` is run.

Reply:
it's in the Makefile already. Unless I've added it in the wrong place

Reply:
Sorry, I must have overlooked that one. My bad.

Reply:
np =)